ext/tokenizer/tokenizer_data.c

```diff
@@ -86,10 +37,25 @@ void tokenizer_register_constants(INIT_FUNC_ARGS) {
 	REGISTER_LONG_CONSTANT("T_CONSTANT_ENCAPSED_STRING", T_CONSTANT_ENCAPSED_STRING, CONST_CS | CONST_PERSISTENT);
 	REGISTER_LONG_CONSTANT("T_STRING_VARNAME", T_STRING_VARNAME, CONST_CS | CONST_PERSISTENT);
 	REGISTER_LONG_CONSTANT("T_NUM_STRING", T_NUM_STRING, CONST_CS | CONST_PERSISTENT);
+	REGISTER_LONG_CONSTANT("T_INCLUDE", T_INCLUDE, CONST_CS | CONST_PERSISTENT);
```

Comment:
What's the new sorting order based on?

Reply:
I just run the `tokenizer_data_gen.sh` script ^^ as I thought that everyone does this.. But then I realized there is a quite some changes... I'm ok to revert this if this script shouldn't be used.

Reply:
I think someone may have changed the order of the tokenizer constants without being aware of this script. It may be useful to run that script in a separate PR anyway if we plan to keep using it.

I didn't notice that that file was created by a script

Reply:
I think they may have been reordered in b03cafd19c01db57b89727ce77cc89a7d816077c which didn't update tokenizer_data.c

Reply:
I'll create a separate PR for claiming back the original token order.